src/me/ryanhamshire/PopulationDensity/DropShipTeleporter.java

```diff
@@ -52,21 +52,13 @@ public void onEntityToggleFlight(EntityToggleGlideEvent event)
             event.setCancelled(true);
         }
     }
-   
-    @EventHandler(ignoreCancelled = true)
-    public void onPlayerMove(PlayerMoveEvent event) {
```

Comment:
Any reason why you removed this event handler? @Developaws added this in https://github.com/TechFortress/PopulationDensity/pull/40 (also this is a good reason why one should title PRs properly, took me a while to find the PR lol)

Reply:
this was the cause of the damage still being present when teleporting to a region

Reply:
Hmm, so playermoveevent would fire when player is on ground but before receiving damage? Makes sense then; PR #40 then is flawed :p